README.rst

```diff
@@ -38,7 +38,7 @@ Example usage
 This example uses the ``dwave.gate.Circuit`` object's  context manager to append operations to
 a two-qubit circuit.
 
-.. testcode::
+.. code-block:: python
```

Comment:
@JoelPasvolsky This was changed _only_ in the readme in root, since the `testcode` block doesn't seem to render on GitHub. I've left it in the docs readme, since it's required for the doctests to pass, although it's not optimal to have two slightly different readme in root/docs.